features/admin.actions.v1/components/action-config-form.tsx

```diff
@@ -0,0 +1,841 @@
+        if (initialValues?.id === undefined) {
+            setIsAuthenticationUpdateFormState(true);
+        } else {
+            setAuthenticationType(initialValues.authenticationType as AuthenticationType);
+            setIsAuthenticationUpdateFormState(false);
+        }
+    }, [ initialValues ]);
+
+    const renderInputAdornmentOfSecret1 = (): ReactElement => (
+        <InputAdornment position="end">
+            <Icon
+                link={ true }
+                className="list-icon reset-field-to-default-adornment"
+                size="small"
+                color="grey"
+                name={ !isShowSecret1 ? "eye" : "eye slash" }
+                data-componentid={ `${ _componentId }-authentication-property-secret1-view-button` }
+                onClick={ () => { setIsShowSecret1(!isShowSecret1); } }
+            />
+        </InputAdornment>
+    );
+
+    const renderInputAdornmentOfSecret2 = (): ReactElement => (
+        <InputAdornment position="end">
+            <Icon
+                link={ true }
+                className="list-icon reset-field-to-default-adornment"
+                size="small"
+                color="grey"
+                name={ !isShowSecret2 ? "eye" : "eye slash" }
+                data-componentid={ `${ _componentId }-authentication-property-secret2-view-button` }
+                onClick={ () => { setIsShowSecret2(!isShowSecret2); } }
+            />
+        </InputAdornment>
+    );
+
+    const renderLoadingPlaceholders = (): ReactElement => {
+
+        return (
+            <>
+                <Skeleton variant="rectangular" className="placeholder label" />
+                <Skeleton variant="rectangular" className="placeholder text-field with-max-width" />
+                <Skeleton variant="rectangular" className="placeholder label" />
+                <Skeleton variant="rectangular" className="placeholder text-field with-max-width" />
+            </>
+        );
+    };
+
+    /**
+     * This is called when the Change Authentication button is pressed.
+     */
+    const handleAuthenticationChange = () : void => {
+        // setIsAuthUpdating(true);
+        setIsAuthenticationUpdateFormState(true);
+    };
+
+    /**
+     * This is called when the cancel button is pressed.
+     */
+    const handleAuthenticationChangeCancel = () : void => {
+        setAuthenticationType(initialValues?.authenticationType as AuthenticationType);
+        setIsAuthenticationUpdateFormState(false);
+    };
+
+    const handleSuccess = (operation: string) => {
+        dispatch(
+            addAlert({
+                description: t("console:manage.features.actions.notification.success." + operation + ".description"),
+                level: AlertLevels.SUCCESS,
+                message: t("console:manage.features.actions.notification.success." + operation + ".message")
+            })
+        );
+    };
+
+    const handleError = (error: AxiosError, operation: string) => {
```

Comment:
Add any missing return types in other files as well.

```suggestion
    const handleError = (error: AxiosError, operation: string): void => {
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/c514505d74137c56840758c44975197ff519db64